SEC/AllSEC.rdf

```diff
@@ -54,7 +56,7 @@
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/SEC/AllSEC/">
 		<rdfs:label>Securities Domain</rdfs:label>
 		<dct:abstract>The FIBO Securities (SEC) domain provides a model of concepts that are common to financial instruments that are also securities, including but not limited to exchange-traded securities. High-level concepts relevant to securities classification, identification, issuance, and registration of securities generally are covered, as well as additional detail for equities, debt instruments, and funds.  More details defining derivatives in particular are covered in a separate derivatives domain area.</dct:abstract>
-		<dct:issued rdf:datatype="&xsd;dateTime">2021-06-28T18:00:00</dct:issued>
+		<dct:issued rdf:datatype="&xsd;dateTime">2022-03-31T18:00:00</dct:issued>
```

Comment:
I don't understand why we're quoting a date in the future! Note that this ontology is "issued" i.e. made publicly available (in both GitHub and the EDMC published version) as soon as it's merged.